app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailConfig.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  TextContent,
+  TextList,
+  TextListItem,
+  TextListItemVariants,
+  TextListVariants,
+} from '@patternfly/react-core';
+import * as React from 'react';
+import { Container } from '../../../Layout';
+import { IApiConnectorDetailValues } from './ApiConnectorDetailBody';
+
+export interface IApiConnectorDetailConfig {
+  i18nLabelBaseUrl: string;
+  i18nLabelDescription: string;
+  i18nLabelHost: string;
+  i18nLabelName: string;
+
+  /**
+   * Initial values displayed,
+   * typically set when creating the connector
+   */
+  properties: IApiConnectorDetailValues;
+}
+
+export const ApiConnectorDetailConfig: React.FunctionComponent<IApiConnectorDetailConfig> = ({
+  i18nLabelBaseUrl,
+  i18nLabelDescription,
+  i18nLabelHost,
+  i18nLabelName,
+  properties,
+}) => {
+  return (
+    <>
+      <TextContent data-testid={'api-connector-detail-config'}>
+        <Container>
```

Comment:
Same here.  I just peeked and yeah, this component just wraps it's content in a div with the `container-fluid` class, I don't think that's part of patternfly anymore.